modules/gdnative/gdnative.cpp

```diff
@@ -430,6 +433,10 @@ bool GDNative::terminate() {
 			gdnatives->clear();
 			// whew this looks scary, but all it does is remove the entry completely
 			GDNativeLibrary::loaded_libraries.erase(GDNativeLibrary::loaded_libraries.find(library->get_current_library_path()));
+		} else {
+			// in case user triggers a call to terminate in the callback below
```

Comment:
```suggestion
			// In case user triggers a call to terminate in the callback below.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).